api/MediaSource.json

```diff
@@ -11,7 +11,7 @@
             {
               "version_added": "23",
               "version_removed": "31",
-              "prefix": "-webkit-"
+              "prefix": "webkit"
```

Comment:
Spot checking this raised a couple of questions, because I found stuff like this: https://bugs.chromium.org/p/chromium/issues/detail?id=239485

1. I'm wondering if the top-level feature and the constructor prefixes should be `WebKit` instead of `webkit`. Seems like the capitalized version is preferred for those

Reply:
FYI. This is an exception to how prefixes was/are spelled in Chromium.